Content.Shared/Preferences/JobPriority.cs

```diff
@@ -1,6 +1,6 @@
 namespace Content.Shared.Preferences
 {
-    public enum JobPriority
+    public enum JobPriority : byte
```

Comment:
As above DB stuff, not sure how it'll go.

Reply:
OK, i will undo this modification.